assets/training/model_management/src/azureml/model/mgmt/config.py

```diff
@@ -29,6 +29,7 @@ class ModelFramework(_CustomEnum):
 
     HUGGINGFACE = "Huggingface"
     MMLAB = "MMLab"
+    PYFUNC_LLAVA = "pyfunc_llava"
```

Comment:
Just thinking out loud, will it be better if we have name like "Microsoft_Internal". I assume it's our internal model.

Reply:
This is non-descriptive wrt the LLaVA model. Future PRs should change the name, as they will have more context than the current PR regarding what a good abstraction is.